pkgs/games/freeciv/default.nix

```diff
@@ -79,7 +78,6 @@ stdenv.mkDerivation rec {
       SDL2_image
       SDL2_ttf
       SDL2_gfx
-      libX11 # can be removed when https://github.com/freeciv/freeciv/commit/00a9d69fa38d8618893e1eaad2830abb395ef25b is tagged, likely in 3.2.0
```

Comment:
Confirmed included in this release as https://github.com/freeciv/freeciv/commit/a73590ea2cfdd8c98342ed8b44e0d3050a0fd7b1